docs/architecture/adr-001-poap-contract.md

```diff
@@ -146,6 +147,16 @@ The metadata should be filled as the following ones:
 ```
 * The `cw721_code_id` identifies the code of the `CW721-base` contract initialised by this contract.
 
+#### TokenExtInfo
```

Comment:
I think we can name this simply as Claimer

Reply:
Maybe we should name it into `Extension`, since the field name is `extension` by querying, like the sample of [cw721-metadata](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-metadata-onchain).
What do you think?

Reply:
What if we name it `Metadata` and we put inside the object the `claimer`?
So that if we need to extend the object with further data it will be easier since we already have the on-chain metadata. What do you think?

Reply:
@bragaz Great, `Metadata` would be more proper.